Formula/p/prometheus.rb

```diff
@@ -7,7 +7,7 @@ class Prometheus < Formula
 
   livecheck do
```

Comment:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the Git tags.
  livecheck do
```

The `stable` URL is a tag archive (instead of a release asset), so this could use a comment to explain the use of the `GithubLatest` strategy.